starter_code/express-spotify-app/views/artists.hbs

```diff
@@ -0,0 +1,14 @@
+
+{{#each artists.items}}
+<div class="artist-card">
+  {{#each this.images}}
+    {{#if @first}}
+      <div class="albumCover">
+        <img src="{{this.url}}" alt="cover">
+      </div>
+    {{/if}}
+  {{/each}}
+  <h2>{{this.name}}</h2>
+  <a class="btn btn-danger" href="/albums/{{this.id}}">Show albums</a>
```

Comment:
I get that you liked the `danger` color, but it's a better practice to use semantic classes, so use `danger` for something that is actually a danger for the user. You could always customize the colors :)
Not a big deal, but just giving some extra tips since you are doing well with the basics